js/web/components/HubReleases.js

```diff
@@ -0,0 +1,239 @@
+                        />
+                      )}
+                    </ResponsivePlayButton>
+                  </StyledTableCell>
+                  <StyledTableCell align="left">
+                    <Box sx={{ width: '50px' }}>
+                      <Image
+                        height={'100%'}
+                        width={'100%'}
+                        layout="responsive"
+                        src={getImageFromCDN(
+                          release.image,
+                          400,
+                          new Date(Date.parse(release.properties.date))
+                        )}
+                        alt={release.name}
+                        priority={true}
+                        loader={loader}
+                      />
+                    </Box>
+                  </StyledTableCell>
+                  <StyledTableCell sx={{ maxWidth: '20vw' }} align="left">
+                    <Box
+                      sx={{
+                        overflow: 'hidden',
+                        textOverflow: 'ellipsis',
+                      }}
+                    >
+                      <Typography noWrap>
+                        {release.properties.artist}{' '}
+                      </Typography>
+                    </Box>
+                  </StyledTableCell>
+                  <StyledTableCell
+                    sx={{ textDecoration: 'underline', maxWidth: '20vw' }}
+                    align="left"
+                  >
+                    <Box
+                      sx={{
+                        overflow: 'hidden',
+                        textOverflow: 'ellipsis',
+                      }}
+                    >
+                      <Typography noWrap>{release.properties.title}</Typography>
+                    </Box>
+                  </StyledTableCell>
+                </TableRow>
+              </Link>
+            ))}
+            <TableRow sx={{ borderBottom: 'none' }}>
+              <StyledTableCell
+                sx={{ height: '50px', borderBottom: 'none' }}
+              ></StyledTableCell>
+            </TableRow>
+          </TableBody>
+        </Table>
+      </TableContainer>
+    </ResponsiveContainer>
+  )
+}
+
+const HubTableHead = ({ tableCategories }) => {
+  return (
+    <TableHead sx={{}}>
+      <TableRow sx={{ py: 1 }}>
+        {tableCategories.map((category) => (
+          <StyledTableCell
+            align="left"
+            key={category}
+            sx={{
+              fontWeight: 'bold',
+              borderBottom: 'none',
+            }}
+          >
+            <Typography sx={{ fontWeight: 'bold' }}>{category}</Typography>
+          </StyledTableCell>
+        ))}
+      </TableRow>
+    </TableHead>
+  )
+}
+
+const StyledTableCell = styled(TableCell)(({ theme }) => ({
+  padding: '5px 0',
+
+  [theme.breakpoints.down('md')]: {
+    padding: '0 5px',
+  },
+}))
+
+const ResponsiveContainer = styled(Box)(({ theme }) => ({
+  width: '960px',
```

Comment:
this width is being reused in a few of your next components, if should be added to the theme and re-used so that its easy to tweak